### PR TITLE
Use .tar.gz instead of .tgz extension

### PR DESCRIPTION
**Pull request by hs-apotell**
Use .tar.gz instead of .tgz extension

Python's tar module messes things up with the .tgz extension making
unarchiving difficult due to the same file name used at multiple levels
of the folder hierarchy. Using .tar.gz (as recommended by the
documentation) seems to work as expected.

### CODE REVIEW

File: .github/workflows/main.yml
Comment (hzeller) [inline .github/workflows/main.yml:113]: the file that is packed up then is named `*.tgz` again, wouldn't we want it name `*.tar.gz` as well ?

Comment (hs-apotell) [reply .github/workflows/main.yml:113]: Those are not archived using python. They are archived using GitHub action.

Comment (hzeller) [reply .github/workflows/main.yml:113]: Yes, but *.tar.gz is the correct naming anyway. *.tgz is a very uncommon way to name such archives.
Also, uniformly names are also beneficial as it leaves people less scratching their heads.

Comment (hs-apotell) [reply .github/workflows/main.yml]: Updated all references of .tgz to .tar.gz.

**Review by hzeller — APPROVED** [review]
LGTM

Let's wait for the CI to finish before merge.

**Pull request merged**
